.github/workflows/cri_test.yml

```
@@ -68,3 +68,49 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/github_runner/clean_cri_runner.sh
+
+  gvisor-cri-tests:
+    name: gVisor-CRI tests
+    env:
+      GITHUB_RUN_ID: ${{ github.run_id }}
+      GITHUB_VHIVE_ARGS: "-dbg"
+    runs-on: [self-hosted, cri]
+
```

please try on a non self-hosted runner instead
```suggestion
    runs-on: [ubuntu-18, cri]

```